.github/workflows/storybook.yaml

```diff
@@ -13,8 +13,19 @@ jobs:
           persist-credentials: false
       - name: Install and Build
```

Comment:
```suggestion
      - name: Install and Build # TODO: Because these can be run separately, we could look at seperating these into jobs, and make one Deploy job that waits for the build jobs
```